src/Api/like.ts

```diff
@@ -0,0 +1,35 @@
+import api from './api';
+import { getAxiosHeader } from './user';
+
+const callCreateLikeAPI = async (callCreateLikeAPIBody: {
+  postId: string | undefined;
+}) => {
```

Comment:
```suggestion
const callCreateLikeAPI = async (callCreateLikeAPIBody: {
  postId?: string;
}) => {
```

ask; 
postId가 없는 경우도 있어?? postId는 api 호출할 때 필수 값 아닌가(잘모름)? undefined을 쓴 배경이 궁금합니당

Reply:
아 recoilLikeState에 atom생성할때 LikeProps | null 이렇게 타입을 지정하고 default를 null로 해놔서 그랬었네요..! 빈문자열이랑 빈배열로 초기화해주고 저 부분 undefined 지워줬습니다!
```js
// Api/like.ts
const callCreateLikeAPI = async (callCreateLikeAPIBody: { postId: string }) => {
  ...
};

// recoilLikeState.ts
export const likeState = atom<LikeProps>({
  key: 'likeState',
  default: {
    id: '',
    likes: [],
  },
});
```